source/libs/get-default-branch.ts

```diff
@@ -22,11 +22,11 @@ function parseBranchFromDom() {
 	}
 
 	// Parse the infobar
-	const [, branchName = undefined] = branchInfo.textContent.trim().match(branchInfoRegex) || [];
+	const [, branchName = undefined] = branchInfo.textContent!.trim().match(branchInfoRegex) || [];
```

Comment:
I think the `!` should be on `branchInfo`, not `textContent`.

Reply:
It's there because `textContent` can be null. See https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent